### PR TITLE
Fetch interceptor additions

### PR DESCRIPTION
**Pull request by msutkowski**
- Renames Ismophric->Isomorphic
- Adds a browser preset
- Updates README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ This library monkey-patches the following native modules:
 - `http.get`/`http.request`
 - `https.get`/`https.request`
 - `XMLHttpRequest`
+- `fetch`
```
Comment (msutkowski) [inline README.md:37]: This is most likely wrong :facepalm:

File: src/createInterceptor.ts [hunk 2]
```diff
@@ -14,32 +14,38 @@ export type Observer = StrictEventEmitter<InterceptorEventsMap>
...
+// Support legacy types so this is not a breaking change
+
+export interface IsomorphicResponse {
   status: number
   statusText: string
   headers: Headers
   body?: string
 }
 
+// Support legacy types so this is not a breaking change?
```
Comment (msutkowski) [inline src/createInterceptor.ts:33]: Just here as a note for review

**Review by msutkowski — COMMENTED** [review]
Would it be worth splitting out examples of browser usage vs node usage being that the original PR has the intent of making that possible?

File: src/createInterceptor.ts (hunk 2 above)
Comment (kettanaito) [reply src/createInterceptor.ts:33]: No need, this hasn't been released yet.

**Review by kettanaito — APPROVED** [review]
Thank you for being my editor for all this time 🙇

**Pull request merged**